docs/release_notes/5.0.23-20.md

```diff
@@ -0,0 +1,22 @@
+# Percona Server for MongoDB 5.0.23-20 (2023-12-21)
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 5.0.23-20 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for [MongoDB 5.0.23 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.23---nov-27--2023).
+
+It supports MongoDB 5.0.23 protocols and drivers.
+
+## Release Highlights
```

Comment:
[AWS IAM authentication](https://github.com/percona/psmdb-docs/aws-iam.md) is now generally available, enabling you to use this functionality in production environments.